src/richie/apps/core/templates/richie/dashboard.html

```diff
@@ -0,0 +1,26 @@
+{% extends "richie/single_column.html" %}
+{% load cms_tags i18n %}
+
+{% block head_title %}
+    {% trans "dashboard" %}
+{% endblock head_title %}
+
+{% block meta_opengraph_contextuals %}
+    {% trans "dashboard" as page_title %}
+    <meta property="og:title" content="{{ page_title |truncatechars:65 }}">
+    <meta property="og:url" content="{{ SITE.web_url }}-{{ page_title }}">
+{% endblock meta_opengraph_contextuals %}
+
+
+{% block subheader_content %}
+    {% trans "dashboard" as page_title %}
+<div class="subheader__container">
+    <h1 class="subheader__title">
+        {{ page_title|title }}
+    </h1>
+</div>
+{% endblock subheader_content %}
```

Comment:
Not sure, this relevant to override the `subheader_content` to display the Dashboard name.
I think we should not displayed anything here in order to display all we want about the Dashboard (breadcrumbs, current page title) through the react app.

So instead to append content within `subheader_content` block, I would override `subheader` block to display nothing at all.

```suggestion
{% block subheader %}{% endblock %}
```